libs/libavro/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# libavro - Makefile for Apache AvroC library
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=avro-c
```

Comment:
If I change the PKG_NAME to "libavro" then I can't use that variable name anywhere else.  It seems to me a change like this would break from convention everywhere else.

now:
PKG_NAME:=avro-c
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz

should be:??
PKG_NAME:=libavro
PKG_SOURCE:=avro-c-$(PKG_VERSION).tar.gz

Or should I rename everything to be "avro" or "avro-c"?  I guess I am unclear what I is desired here before this gets merged.  Please advise.